src/constants/index.js

```diff
@@ -1,6 +1,6 @@
 import { keywords } from '../emoji-data';
 import keywordsSingleChar from '../SearchBar/helpers/init_keywords_single';
-import * as scss from '../base.scss';
+import * as scss from '../base.scss'; // eslint-disable-line no-unused-vars
```

Comment:
eslint was complaing about scss not being used. I'm assuming this was intentional so I just suppressed the warning. Happy to remove if that makes sense.